configs/data/all_transforms/DropoutObjects.yaml

```diff
@@ -0,0 +1,7 @@
+DropoutObjects:
+  _target_: tcn_hpl.data.components.augmentations.DropoutObjects
+  skip_stride: 3
```

Comment:
This is a new variable, right? `skip_stride` = the max amount of sequential frames for which dropout might happen in augmentation IIUC?

Reply:
Since this is not being used for now, we can ignore for now